app/pods/node/model.js

```diff
@@ -0,0 +1,36 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  /**
+   * Cluster the node belongs to.
+   * @property cluster
+   * @type DS.Model
+   */
+  cluster: DS.belongsTo('cluster', { async: true }),
+
+  /**
+   * Whether or not the node is available when pinged.
+   *
+   * @property available
+   * @type Boolean
+   */
+  available: DS.attr('boolean', { default: false }),
+
+  /**
+   * All of the nodes configuration settings, stored as an array of key-value objects.
```

Comment:
We need the configs as an object instead of an array (meaning, as a hashmap or dictionary). 
Although we need it as an array to display on the page, that's a rare use case, whereas we'll be referencing the configs fairly frequently (like, is Search on, for this node? etc). 
So, keep the configs as a hashmap (plain JS object), and turn it into an array for displaying on that page.
The Bucket Type properties solves this same case, look at `BucketType.propsList()`